lib/src/async_environment.dart

```diff
@@ -467,6 +512,19 @@ class AsyncEnvironment {
       return;
     }
 
+    if (_nestedForwardedModules != null &&
+        !_variableIndices.containsKey(name) &&
+        _variableIndex(name) == null) {
+      for (var modules in _nestedForwardedModules.reversed) {
+        for (var module in modules.reversed) {
+          if (module.variables.containsKey(name)) {
+            module.setVariable(name, value, nodeWithSpan);
```

Comment:
Yes, this is intended. If you `@import` a forwarded library, you should be able to set that library's variables (just like you'd be able to if you `@use`d it, or if you `@importe`ed a library that uses `@import` instead of `@forward`). See for example [this spec](https://github.com/sass/sass-spec/blob/3653ded68941a262fe8415d191b4b8dc2f5782fe/spec/directives/forward/member/import.hrx#L134-L152).